tests/util.spec.ts

```diff
@@ -18,6 +18,6 @@ describe("URL Resolve", () => {
 
     test("URL Resolve fails with invalid URL", () => {
         const resolveWrap = () => resolveURL("ht/example", "/test///", "////foo/baz/lul", "/bar///");
-        expect(resolveWrap).toThrowError(new TypeError("Invalid URL: ht/example/test/foo/baz/lul/bar"));
+        expect(resolveWrap).toThrowError(new TypeError("Invalid URL"));
```

Comment:
Your tests are currently running with node 12 in ci to ensure compatibility.
The error message was changed in node 16 (thanks log4shell, i guess)
This seams to work in both versions.

```suggestion
        expect(resolveWrap).toThrowError(new RegExp("Invalid URL.*"));
```